packages/salesforcedx-vscode-core/src/commands/startApexDebugLogging.ts

```diff
@@ -38,7 +41,12 @@ export const turnOnLogging = async (extensionContext: vscode.ExtensionContext):
     extensionContext.workspaceState.update(userSpecificKey, expirationDate);
     showTraceFlagExpiration(expirationDate);
 
-    await handleFinishCommand(command, true);
+    channelService.showCommandWithTimestamp(
+      `${nls.localize(coerceMessageKey('long_command_end'))} ${nls.localize(coerceMessageKey(command))}`
+    );
+
+    await notificationService.showInformationMessage(`${nls.localize(coerceMessageKey(command))} successfully ran`);
+    telemetryService.sendCommandEvent(command);
```

Comment:
Revert after moving this code back into `handleFinishCommand()`.

```suggestion
    await handleFinishCommand(command, true);
```